### PR TITLE
DCOS-15921 (2 of 2): Use ComponentExample for design system checkbox

### PR DESCRIPTION
**Pull request by tnguyen0**
Uses the ComponentExample code preview component to show the React and HTML code
representation of the design system checkbox component.

:warning: This PR depends on: #2347

See branch `tommy/design-system/component-example-ace-editor` for the complete implementation.

In the UI: `Design System` --> `Components` --> `Checkboxes` --> `Code Tab`

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: src/styles/index.less [hunk 1]
```diff
@@ -5,6 +5,7 @@
 @import '../../node_modules/cnvs/styles/cnvs.less'; // Source CNVS
 @import 'vendor/cnvs/styles.less'; // CNVS Overrides
 @import 'vendor/loaders.less';
+@import '../../plugins/design-system/styles/styles.less'; // TODO: Tommy figure out where to include this.
```
Comment (nLight) [inline src/styles/index.less]: What's the problem with that one?

File: plugins/design-system/styles/shame.less [hunk 2]
```diff
@@ -1,5 +1,4 @@
 // Shame CSS
-// A place to put quick CSS iterations and cleanup later
```
Comment (nLight) [inline plugins/design-system/styles/shame.less]: Can we delete this file? In this PR https://github.com/dcos/dcos-ui/pull/2388/files you've introduced a separate file for the `do-dont` component. Doesn't matter how messy css is it should all go there.

File: src/styles/index.less (hunk 1 above)
Comment (tnguyen0) [reply src/styles/index.less]: We want to be able to use CNVS styles for plugins. The point was discussed and it was decided this way is fine. I will remove the comment.

File: plugins/design-system/styles/shame.less (hunk 2 above)
Comment (tnguyen0) [reply plugins/design-system/styles/shame.less]: Yes :)

**Comment by nLight** [comment]
Alfred, retest this please

**Pull request merged**